kolibri/plugins/learn/assets/src/views/classes/AllClassesPage.vue

```diff
@@ -2,7 +2,7 @@
 
   <div>
     <div v-if="isUserLoggedIn ">
-      <h2>{{ coreString('classesLabel') }}</h2>
+      <h1>{{ coreString('classesLabel') }}</h1>
```

Comment:
If this gets to become H1, then the best practice would be to change content card titles to H2 🙂 

![2019-08-03_1-34-12](https://user-images.githubusercontent.com/1457929/62403663-307d9c80-b58f-11e9-92e8-51c93dd3b760.png)